### PR TITLE
SignedData creation without certificates

### PR DESCRIPTION
**Pull request by bilalashraf123**

### CODE REVIEW

File: sign.go [hunk 1]
```diff
@@ -32,7 +32,7 @@ func NewSignedData(data []byte) (*SignedData, error) {
 	return signedData, nil
 }
 
-// One of the practical usecase of using this function is below:
+// One of the practical use case of NewSignedDataWithContentType is:
```
Comment (vanbroup) [inline sign.go]:
```suggestion
// One of the practical use cases of NewSignedDataWithContentType is:
```

File: sign.go [hunk 2]
```diff
@@ -143,7 +143,7 @@ func (sd *SignedData) AddSigner(ee *x509.Certificate, pkey crypto.PrivateKey, co
 	return sd.addSignerChain(ee, pkey, parents, config, true)
 }
 
-// One of the practical usecase of using this function is below:
+// One of the practical use case of AddSignerNoChain is:
```
Comment (vanbroup) [inline sign.go]:
```suggestion
// One of the practical use cases of AddSignerNoChain is:
```

File: sign.go (hunk 1 above)
Comment (bilalashraf123) [reply sign.go]: Done.

File: sign.go (hunk 2 above)
Comment (bilalashraf123) [reply sign.go]: Done.

**Pull request merged**